http-server/src/main/java/de/unistuttgart/iaas/amyassist/amy/httpserver/ThrowableExceptionMapper.java

```diff
@@ -43,6 +51,12 @@
 	public Response toResponse(Throwable t) {
 		if (t instanceof WebApplicationException) {
 			WebApplicationException webEx = (WebApplicationException) t;
+			if(webEx.getResponse().getStatus() == 500) {
```

Comment:
all exceptions must be logged on server side, because clients often don't show the exceptions to user or handle them and for debugging its important to have the full stack trace

Reply:
nope... das ist absicht alles was keine 500 ist ist kein fehler

Reply:
Ja ich finde auch nur die 500er Exceptions sollten geloggt werden.